chrome/content/zotero/xpcom/openurl.js

```diff
@@ -407,7 +407,7 @@ Zotero.OpenURL = new function() {
 				
 				item.creators.push(_cloneIfNecessary(Zotero.Utilities.cleanAuthor(value, type, value.indexOf(",") !== -1), item));
 			} else if(key == "rft.aucorp") {
-				complexAu.push(_cloneIfNecessary({lastName:value, isInstitution:true}, item));
+				complexAu.push(_cloneIfNecessary({lastName:value, fieldMode:1}, item));
```

Comment:
This one might be correct, though, if it's meant to be returning internal JSON (which is what "the format returned by item.fromArray()" basically means).